generators/server/templates/src/main/java/package/config/_MicroserviceSecurityConfiguration.java

```diff
@@ -121,9 +129,27 @@ public TokenStore tokenStore() {
       @Bean
       public JwtAccessTokenConverter jwtAccessTokenConverter() {
           JwtAccessTokenConverter converter = new JwtAccessTokenConverter();
-          converter.setSigningKey(jHipsterProperties.getSecurity().getAuthentication().getJwt().getSecret());
-
+          converter.setVerifierKey(getKeyFromAuthorizationServer());
           return converter;
       }
+
+      @Bean
+      @Qualifier("LoadBalancedRestTemplate")
+      public RestTemplate restTemplate(RestTemplateCustomizer customizer) {
+          RestTemplate restTemplate = new RestTemplate();
+          customizer.customize(restTemplate);
+          return restTemplate;
+      }
+
+      @Inject
+      @Qualifier("LoadBalancedRestTemplate")
+      private RestTemplate keyUriRestTemplate;
```

Comment:
I don't understand, you create a Bean just on line 137, to inject it only here? I think this could be easier